sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernatePrinterDao.java

```diff
@@ -110,7 +110,12 @@ public String propertyForDate(Criteria criteria, DateType type) {
 
   @Override
   public String propertyForSortColumn(String original) {
-    return original;
+    switch (original) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)